example/windows/CMakeLists.txt

```diff
@@ -0,0 +1,101 @@
+# Project-level configuration.
```

Comment:
<rant>
oh god, cmake

i just hate it, why couldn't flutter push for Gradle for windows/linux as well haha
</rant>

Reply:
haha, just generated code. I think it would be too much work to change this ;-)